packages/common/src/api/README.md

```diff
@@ -0,0 +1,150 @@
+# audius-query
```

Comment:
TODO: I _think_ github automatically makes a table of contents, but if not, I'll add one after

Reply:
I don't think it does unfortunately